packages/form-section/src/donation-form-section.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  CSSResult,
+  TemplateResult,
+  property,
+} from 'lit-element';
+
+export enum DonationFormSectionNumberMode {
+  HideNumber = 'hidenumber',
+  ShowNumber = 'shownumber',
+}
+
+@customElement('donation-form-section')
+export class DonationFormSection extends LitElement {
+  @property({ type: Number }) number = 0;
+
+  @property({ type: String }) headline: string | undefined;
+
+  @property({ type: String }) numberMode: DonationFormSectionNumberMode =
+    DonationFormSectionNumberMode.ShowNumber;
+
+  /** @inheritdoc */
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <div class="number-container">
+          <div class="number">${this.number}</div>
+        </div>
+        <div class="content-container ${this.contentContainerStyle}">
```

Comment:
Could we just use the enum value as the class name here and toggle accordingly? That would eliminate the need for a getter for this toggled class.